### PR TITLE
Add changelog and release schedule

### PR DESCRIPTION
**Pull request by Abijeet**
Divide the changelog into - Language updates, PHP and Node.js section

PHP and Node.js changelog will list library related changes and will
reuse Language updates section.

Add the release schedule to the README file.

Bug: T218639

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+5. [Add Sylheti ](https://github.com/wikimedia/language-data/pull/94)
+6. [Split ary to ary-latn and ary-arab](https://github.com/wikimedia/language-data/pull/92)
+7. [Add Middle East and Africa to relevant Arabic varieties](https://github.com/wikimedia/language-data/pull/92)
+8. [Add Mongolian in vertical script ](https://github.com/wikimedia/language-data/pull/95)
+9. [Add AM as a region for Venetian](https://github.com/wikimedia/language-data/pull/96)
+
+## PHP
+### PHP 1.0.0 (2020-05-20)
+- See [language updates 2020-05-20](#2020-05-20).
+- Add PHP support. Documentation is [here](https://language-data.readthedocs.io/en/latest/api/languagedata/languageutil.html). Packagist is [here](https://example.com).
```
Comment (Nikerabbit) [inline CHANGELOG.md:25]:
```suggestion
- Add PHP support. [Documentation](https://language-data.readthedocs.io/en/latest/api/languagedata/languageutil.html). [Packagist](https://packagist.org/packages/wikimedia/language-data).
```

File: README.md [hunk 2]
```diff
@@ -32,4 +32,12 @@ This data is populated from the current version of
...
 [node-build-url]: https://github.com/wikimedia/language-data/actions?query=workflow%3A%22Node.js+build%22
 [php-build]: https://github.com/wikimedia/language-data/workflows/PHP%20build/badge.svg
-[php-build-url]: https://github.com/wikimedia/language-data/actions?query=workflow%3A%22PHP+build%22
\ No newline at end of file
+[php-build-url]: https://github.com/wikimedia/language-data/actions?query=workflow%3A%22PHP+build%22
+
+## Release schedule
+Similar to [MLEB](https://www.mediawiki.org/wiki/MediaWiki_Language_Extension_Bundle), this library
+will have a quarterly release schedule, and will be released along with MLEB. Intermediate releases
+will be made incase a major bug is identified.
```
Comment (Nikerabbit) [inline README.md:40]:
```suggestion
will be made for important bug fixes.
```

File: README.md [hunk 3]
```diff
@@ -32,4 +32,12 @@ This data is populated from the current version of
...
+
+## Changelog
+The full changelog is available [here](https://github.com/wikimedia/language-data/blob/master/CHANGELOG.md).
```
Comment (Nikerabbit) [inline README.md:43]:
```suggestion
The full changelog is available in [CHANGELOG.md](https://github.com/wikimedia/language-data/blob/master/CHANGELOG.md).
```

**Comment by Abijeet** [comment]
When merging, please squash and merge.

**Pull request merged**